conduit.go

```diff
@@ -79,7 +79,7 @@ var ConnectService = &cobra.Command{
   cf conduit mysql-instance -- mysqldump --all-databases > backup.sql
 
   Import a mysql script:
-  cf conduit mysql-instance -- psql < backup.psql
+  cf conduit mysql-instance -- mysql < backup.psql
```

Comment:
The example filename probably shouldn't have a `.psql` extension either.

Reply:
Fixed 👍